easybuild/toolchains/fft/intelfftw.py

```diff
@@ -98,7 +98,9 @@ def _set_fftw_variables(self):
         imklfftwroot = get_software_root('imkl-FFTW')
         if imklfftwroot:
             # only get cluster_interface_lib from seperate module imkl-FFTW, rest via libmkl_gf/libmkl_intel
-            fft_lib_dirs += [os.path.join(imklfftwroot, 'lib')]
+            if self.options.get('usempi', False):
```

Comment:
@bartoldeman Why the extra guard?

Reply:
Because if `usempi = False` (see above in code) all the FFTW functionality is taken from MKL core libraries so $FFT_LIB_DIR  should *not* be changed.